index.js

```diff
@@ -6,11 +6,15 @@ var YAML = require('yamljs');
 /**
  *
  * @param {number|string} performanceYear - four digit year
- * @return {Array<Benchmark>} - benchmarks data for given year
+ * @return {Array<Benchmark>|undefined} - benchmarks data for given year
  */
 exports.getBenchmarksData = function(performanceYear) {
-  return JSON.parse(
-    fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  try {
+    return JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  } catch (error) {
+    return;
```

Comment:
@sungchoi i just educated myself on the repo and see that we're already loading files from fs. I think to be consistent, we could:

1. catch file-not-found (`catch(error) { error.code === 'ENOENT'`) errors and rethrow with a more descriptive message like 'no benchmark found for year x'
2. rethrow all other errors

re: second suggestion, looks like we've made the decision to load from fs already, so I think that's fine to stick with!

what do you think?